CMSgov/onemac
# OY2-1116 Package Dashboard Updates - 90th Day Data from SEA Tool

Story: https://qmacbis.atlassian.net/browse/OY2-11116
Endpoint: https://dobeqcg0bhygh.cloudfront.net/

Product Team slides are approved!

### Details
When packages are in review, there is a 90 day clock which starts when the State User submits the package to CMS.  SEA Tool manages the clock, oneMAC shows a 90th day column on the package list which shows: 
- N/A for Terminated/Withdrawn//Unsubmitted statuses
- the clock date for any status but Terminated/Withdrawn//Unsubmitted
- If the clock date is unavailable, Pending is shown

### Changes
- Added the DynamoDB stream ARN that was missing from 10638 (develop was fixed in console)
- Added clockEndTimestamp to the Dynamodb projection in getMyPackages;
- clockEndTimestamp is removed from saving to the package database in submit.js - we do use it in the emails, but lack the authority to call that the "real clock"
- Column added to package list 
- Unit Testing added to handle the status/timestamp triage
- Code Climate finding led to moving the corresponding RAI link to cmscommonlib
- Updated PackageList to use "package" wording instead of "changeRequest"

### Test Plan
Unit Tests test the existence of the 90th Day column and the relationship between status and contents of the 90th day column.
1. Log in as a test account submitting user
2. Submit a package
3. Click Packages link in NAV
4. Verify that the 90th Day is showing and is (probably) Pending for the recently added package. SEA Tool may have updated the status, but at least in DEV, the clock is not yet getting set
5. Withdraw the package
6. Verify the 90th Day now displays N/A
